docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -97,74 +108,41 @@ endpoint will be traced without any required code changes.
 
 === Create the configuration
 
-There are two ways to configure the default OTLP gRPC Exporter within the application.
-
-The first approach is by providing the properties within the `src/main/resources/application.properties` file:
+To configure the default OTLP gRPC Exporter within the application you must set the properties within the `src/main/resources/application.properties` file:
 
 [source,properties]
 ----
 quarkus.application.name=myservice // <1>
-quarkus.opentelemetry.enabled=true // <2>
-quarkus.opentelemetry.tracer.exporter.otlp.endpoint=http://localhost:4317 // <3>
-
-quarkus.opentelemetry.tracer.exporter.otlp.headers=Authorization=Bearer my_secret // <4>
-
-quarkus.log.console.format=%d{HH:mm:ss} %-5p traceId=%X{traceId}, parentId=%X{parentId}, spanId=%X{spanId}, sampled=%X{sampled} [%c{2.}] (%t) %s%e%n  // <5>
+quarkus.otel.exporter.otlp.traces.endpoint=http://localhost:4317/v1/traces // <2>
+quarkus.log.console.format=%d{HH:mm:ss} %-5p traceId=%X{traceId}, parentId=%X{parentId}, spanId=%X{spanId}, sampled=%X{sampled} [%c{2.}] (%t) %s%e%n  // <3>
 
 # Alternative to the console log
-quarkus.http.access-log.pattern="...traceId=%{X,traceId} spanId=%{X,spanId}" // <6>
+quarkus.http.access-log.pattern="...traceId=%{X,traceId} spanId=%{X,spanId}" // <4>
 ----
 
 <1> All spans created from the application will include an OpenTelemetry `Resource` indicating the span was created by the `myservice` application. If not set, it will default to the artifact id.
-<2> Whether OpenTelemetry is enabled or not. The default is `true`, but shown here to indicate how it can be disabled
-<3> gRPC endpoint for sending spans
-<4> Optional gRPC headers commonly used for authentication
-<5> Add tracing information into log message.
-<6> You can also only put the trace info into the access log. In this case you must omit the info in the console log format.
+<2> gRPC endpoint to send spans
+<3> Add tracing information into log message.
+<4> You can also only put the trace info into the access log. In this case you must omit the info in the console log format.
+
+[NOTE]
+====
+All configurations have been updated from `quarkus.opentelemetry.\*` -> `quarkus.otel.*`
```

Comment:
Sorry to ask that but is it really an improvement? What's the rationale of this change?

Reply:
This is to align with the OTel side property semantics. See: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#disabling-opentelemetrysdk

We will transparently support them from now on.

Reply:
When OTel introduced its configuration namespace, we discussed and decided to align with it as close as possible:
- Since the goal of OTel is to be supported across many cloud providers, tools, servers, etc, we want to stay within it. Ideally, users should use the same configuration as what they use in other external OTel aware components
- We cannot prevent users from plugin in the OTel Agent, which uses the OTel configuration as specified. This should unify that configuration, so when you plug in the agent, it will use the defined Quarkus configuration (if any)

Reply:
OK. Thanks for the clarification.
I wonder if we should rewrite the properties as part of the upgrade process. Unfortunately, we cannot use regexps so we will have to map the old names to the new names manually.

Maybe it's not worth the hassle, I don't know.